identity/webapp/src/frontend/MyAccount/MyAccount.tsx

```diff
@@ -44,7 +54,10 @@ const Detail: React.FC<DetailProps> = ({ label, value }) => (
   </>
 );
 
-const AccountStatus: React.FC<React.ComponentProps<typeof StatusAlert>> = ({ type, children }) => {
+const AccountStatus: React.FC<React.ComponentProps<typeof StatusAlert>> = ({
```

Comment:
```suggestion
const AccountStatus: FC<ComponentProps<typeof StatusAlert>> = ({
```

And import those from react?